client/src/Homes.jsx

```diff
@@ -2,37 +2,71 @@ import React from 'react';
 import './style.css';
 import Stars from './Stars';
 
-const Homes = (props) => {
-  const { data } = props;
-  const listing = data.map((home) => {
+const Homes = ({ handleNext, data, prev, next, handlePrev }) => {
+  const listing = data.map(home => (
```

Comment:
may want to de-structure home here--just a thought!

Reply:
nice catch, will implement, thank you!